### ISSUE TITLE
[Question] Unable to open slide in nuxt!

### ISSUE THREAD

**Issue by e0lithic** [alvarotrigo/vue-fullpage.js#159]
Hi, I am just getting my hands wet in frontend developement and came across your brilliant library and purchased a hobby license for the same. I followed the guide for using your library on nuxt. However, whenever I am trying to open the second slide on  the third section it gives me the following error .
`Failed to execute 'querySelector' on 'Document': '#page3/1' is not a valid selector.`

### Code
```js
<template>
  <div>
    <full-page ref="fullpage" :options="options" id="fullpage">
      <div class="section">
        First section ...
      </div>
      <div class="section">
        Second section ...
      </div>
      <div class="section">
        <div class="slide">
          Slide 1
        </div>  
        <div class="slide">
          Slide 2
        </div>  
      </div>
    </full-page>
  </div>
</template>

<script>
  export default {
    data () {
      return {
        options: {
          licenseKey: 'OPEN-SOURCE-GPLV3-LICENSE',
          afterLoad: this.afterLoad,
          // scrollBar: false,
          // menu: '#menu',
          navigation: true,
          anchors: ['page1', 'page2', 'page3'],
          sectionsColor: ['#41b883', '#ff5f45', '#0798ec', '#fec401', '#1bcee6', '#ee1a59', '#2c3e4f', '#ba5be9', '#b4b8ab']
        }
      }
    },
    methods: {
      afterLoad () {
      }
    }
  }
</script>
```

### Versions 
OS- Ubuntu - 18.04
Chrome - Version 79.0.3945.79 (Official Build) (64-bit)
nuxt-fullpage.js - 0.0.3

Any pointers would be greatly appreciated.

**Comment by alvarotrigo** [comment]
Have you read the docs regarding the usage with Nuxt?
https://github.com/alvarotrigo/vue-fullpage.js#usage-with-nuxtjs

**Comment by e0lithic** [comment]
Yup, I read through the docs, and I am exactly followed the method involving nuxt-fullpage.js plugin.
For the sections the plugin seems to be working flawlessly and the vertically scrolling works great. However, when I try to move through the horizontal slide , it gives me the error I mentioned above.

**Comment by alvarotrigo** [comment]
I would recommend you to look at the nuxt example using `nuxt-fullpage.js`:
https://github.com/vaso2/nuxt-fullpage.js/tree/master/example

**Comment by e0lithic** [comment]
Thanks for the quick reply. I have infact used the very example you mentioned for the code I have pasted above. 

Another thing to note , is that when I don't mention the anchor i.e
`anchors: ['page1', 'page2', 'page3']`, in that case the horizontal slides work as expected ( no change in urls for the sections or slides). However as soon as the anchors are mentioned and the routing urls start using the #section_name followed by the /slide_name, that it gives me the error. I tried using the data-anchor as well , but to no avail. So - 
`localhost:3000/test_fullpage#page3` works but `http://localhost:3000/test_fullpage#page3/2` doesn't. I apologize for the basic queries.

**Comment by alvarotrigo** [comment]
That sounds like it might be an issue in the nuxt-fullpage.js project, can you please create an issue in it to report it?
https://github.com/vaso2/nuxt-fullpage.js/issues

**Comment by alvarotrigo** [comment]
@vaso2 any idea about why is this happening?

**Comment by denyo256** [comment]
I just started a new vue-fullpage.js and got the same problems. As soon as I use the anchors option, scrolling through the sections behaves strange and scrolling to slides on these sections isn't possible at all. 
The url changes from http://localhost/myproject/#page3/2 to http://localhost/myproject/#page3 all the time
